lib/index.js

```diff
@@ -7,8 +7,9 @@ const pkg = JSON.parse(fs.readFileSync(new URL('../package.json', import.meta.ur
 const url = `https://raw.githubusercontent.com/imagemin/cwebp-bin/v${pkg.version}/vendor/`;
 
 const binWrapper = new BinWrapper()
-	.src(`${url}osx/cwebp`, 'darwin')
-	.src(`${url}linux/x86/cwebp`, 'linux', 'x86')
+	.src(`${url}osx/arm64/cwebp`, 'darwin', 'arm64')
+	.src(`${url}osx/x86-64/cwebp`, 'darwin', 'x86-64')
```

Comment:
I didn't see any documentation in bin-wrapper about recognized architectures. These are how they're referenced in macOS and Linux, but I'm happy to change them to x64 and arm64 to match if that will work.